plugins/action_refresh_review_assignments/src/refreshReviewAssignments.ts

```diff
@@ -0,0 +1,68 @@
+import { ActionPluginInput } from '../../types'
+import databaseMethods from './databaseMethods'
+import { ActionQueueStatus } from '../../../src/generated/graphql'
+import generateReviewAssignments from '../../action_generate_review_assignment_records/src/generateReviewAssignments'
+
+async function refreshReviewAssignments({
+  parameters,
+  applicationData,
+  DBConnect,
+}: ActionPluginInput) {
+  const db = databaseMethods(DBConnect)
+
+  // If no userId parameter, we will update ALL active applications
+  const shouldRefreshAll = !parameters?.userId
+
+  // Can handle input of either single userId or an array of userIds
+  const userIds = Array.isArray(parameters?.userId) ? parameters.userId : [parameters.userId]
```

Comment:
Maybe it would throw an error here when parameters.userId is undefined?

Reply:
Okay, I've changed it to put the `?.` operator in all references here. 6a41f29b